babel/index.js

```diff
@@ -2,6 +2,7 @@
 
 var _ = require('lodash'),
     request = require('request'),
+    rpn = require('request-promise-native'),
```

Comment:
Is this definitely needed?

Reply:
https://www.npmjs.com/package/request#promises--asyncawait
If we want to use promises or async / await I think so